### DISCUSSION TITLE
How to config for usage without total power

### DISCUSSION THREAD

**Discussion by adnet-solutions** [emporia-vue-local/esphome#189] [Q&A]
Hi,
I would like to not connect the 3 120A clamps. How i need to modify the yaml file that he not use this main clamps.
I dont need total power and all this. I only need the power of every of that 16 50a clamps as value.

**Comment by flaviut** [comment]
Simply remove those entries & the places they are referenced (i.e. `id(phase_a_power).state`) from the config

**Comment by adnet-solutions** [comment]
i have removed (comment out with #) now all what i think is not needed- But i get a error message.

```
INFO ESPHome 2023.7.1
INFO Reading configuration /config/esphome/emporiavue2.yaml...
WARNING GPIO12 is a Strapping PIN and should be avoided.
Attaching external pullup/down resistors to strapping pins can cause unexpected failures.
See https://esphome.io/guides/faq.html#why-am-i-getting-a-warning-about-strapping-pins
INFO Detected timezone 'Europe/Berlin'
INFO Generating C++ source...
INFO Compiling app...
Processing emporiavue2 (board: esp32dev; framework: espidf; platform: platformio/espressif32@5.3.0)
--------------------------------------------------------------------------------
HARDWARE: ESP32 240MHz, 320KB RAM, 4MB Flash
 - framework-espidf @ 3.40404.0 (4.4.4) 
 - tool-cmake @ 3.16.4 
 - tool-ninja @ 1.7.1 
 - toolchain-esp32ulp @ 2.35.0-20220830 
 - toolchain-xtensa-esp32 @ 8.4.0+2021r2-patch5
Reading CMake configuration...
Dependency Graph
|-- ArduinoJson @ 6.18.5
*** [/data/emporiavue2/.pioenvs/emporiavue2/src/esphome/components/template/sensor/template_sensor.o] Source `src/esphome/components/template/sensor/template_sensor.cpp' not found, needed by target `/data/emporiavue2/.pioenvs/emporiavue2/src/esphome/components/template/sensor/template_sensor.o'.
========================== [FAILED] Took 4.03 seconds ==========================
```

i have only connected one clamp now on Port 16!

how should my configfile look.

i try to attach my config to that post.
.
[emporia-githubpost.txt](https://github.com/emporia-vue-local/esphome/files/12269078/emporia-githubpost.txt)

If i comment all in i get it run but i did not get any real value on Cicuit 16. it if flipping between 20W and 0 W. But the power on that clamp should be around 300 W

**Reply by flaviut** [reply]
> template_sensor.cpp' not found, needed by target

This is an odd error, and it's not related to your config file or the emporia vue integration.

[Can you try clearing your esphome caches?](https://gist.github.com/flaviut/93a1212c7b165c7674693a45ad52c512?permalink_comment_id=4200456#gistcomment-4200456)

**Reply by adnet-solutions** [reply]
hi, okay thats help. thx a lot. at least i can now upload my config. But somehow i dont get any measurements on circuit 16 which is the only clamp which is connected. I have there a Device with 300W connected but ESPHome show still 0.
Any idea?

**Reply by flaviut** [reply]
clamp is probably on backwards. Remove the `, *pos` in the config & you should see a negative number.

**Reply by adnet-solutions** [reply]
Cool thats help. Means i should turn the clamp to the other direction. Right?

**Reply by flaviut** [reply]
If you want, and depending on the situation. There are cases (like solar) where you want negative readings.

There is no loss of precision or other problems having them the wrong way around, and you can also much more easily use the `*invert` filter to flip the reading in software.

**Comment by adnet-solutions** [comment]
ok then i go with the *invert filter.
Thx a lot for your help. At least i see now the value. Just wondering that the difference between the expected and the shown value is so big. I expect now a value from around 560W in the last test but the esphome shows only 311W. Is it really so inaccurate?

**Reply by flaviut** [reply]
> Is it really so inaccurate?

Expected error is a max of *maybe* 5%. 560W -> 311W is too much error.

Is the correct phase selected? Is the clamp fully closed & the connector fully inserted into the Vue?

**Comment by adnet-solutions** [comment]
clamp closed -> yes
connector fully inserted -> yes
The phase i will doublecheck tomorrow and also doublecheck again the first to points.

**Comment by adnet-solutions** [comment]
Hi,
i guess i choosed the wron phase in config for that clamp. its looks like that my electrian did a misstake and connected blue to phase 1 and black to phase 3 in time where he installed my emporia. really thx for your help here.!
